server/test/BraveAlerterConfiguratorTest.js

```diff
@@ -1,261 +0,0 @@
-const { ALERT_STATE, AlertSession } = require('brave-alert-lib')
```

Comment:
Split this file up into a separate file for each function that is being tested. This will hopefully make it easier to find tests and will keep the files from getting too long.